src/python/system/path_patcher.py

```diff
@@ -139,6 +139,11 @@ def patch():
   if not _is_windows():
     return
 
+  # TODO(mbarbella): Verify that this is no longer necessary in Python 3 and
```

Comment:
Yes it seems not needed, see https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation, can rephrase this TODO.

Reply:
Done!